tests/conftest.py

```diff
@@ -134,7 +134,7 @@ def pytest_configure(config):
     global_data.username = username
     global_data.password = password
     global_data.db_name = tst_db_name
-    global_data.db_version = version.parse(db_version)
+    global_data.db_version = version.parse(db_version.split("-")[0])
```

Comment:
just want to highlight this `split`. Let's us test with a nightly build like `3.12.0-NIGHTLY.20240116`